internal/ingestion/component/chunker/tool-py/capture_golden.py

```
@@ -132,15 +132,22 @@ def _assert_tokenizer_alive() -> None:
     probe = "RAGFlow chunker parity tokenizer liveness probe."
-    got = num_tokens_from_string(probe)
+    hint = (
+        "Fix the encoding first: uv run python3 ragflow_deps/download_deps.py "
+        "(provides ragflow_deps/cl100k_base.tiktoken)."
+    )
+    # Two shapes to catch. common/token_utils.py builds the encoder on first use and
+    # lets that failure propagate, so an unavailable BPE table raises here; a failure
+    # to encode an otherwise valid string is still reported as 0.
+    try:
+        got = num_tokens_from_string(probe)
+    except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)